src/index.css

```diff
@@ -1,11 +1,11 @@
 div {
   margin: 5px;
-  max-width: 90%;
+  max-width: 1000px;
```

Comment:
Modifying the css is going to be trickier than this. We can explore the ui-schema prop that can be used in an rjsf Form. You may need to go into deeper granularity to render things the way you want. Something like:

```
body {
  max-width: 80%;
}

div.form-group.field.field-string {
  overflow: visible;
  min-width: 1000px;
  max-width: 80%;
}

div.form-group.field.field-number {
  max-width: 80%;
}

div.form-group.field.field-control {
  max-width: 80%;
}

p.col-xs-offset-9 {
  box-sizing: border-box;
  margin: 5px;
  position: relative;
  min-height: 1px;
  padding-right: 15px;
  padding-left: 15px;
  float: left;
  width: 25%;
}

div {
  margin: 5px;
}
  
h1 {
  color: navy;
  margin: 5px;
}
```